### PR TITLE
test(deconflict): add case of the nested assgin pattern

### PR DESCRIPTION
**Pull request by hyf0**
### Test Plan

---

### CODE REVIEW

**Comment by hyf0** [comment]
Current dependencies on/for this PR:
* main
  * **PR #193** <a href="https://app.graphite.dev/github/pr/rolldown-rs/rolldown/193" target="_blank">Graphite</a> 
    * **PR #194** <a href="https://app.graphite.dev/github/pr/rolldown-rs/rolldown/194" target="_blank">Graphite</a>  👈
      * **PR #195** <a href="https://app.graphite.dev/github/pr/rolldown-rs/rolldown/195" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/rolldown-rs/rolldown/194?utm_source=stack-comment).

**Comment by underfin** [comment]
Look like it need rebase.

**Comment by hyf0** [comment]
> Look like it need rebase.

Done by `gt sync`

File: .vscode/settings.json [hunk 1]
```diff
@@ -31,7 +31,8 @@
     "Renamer",
     "rolldown",
     "rollup",
-    "rustc"
+    "rustc",
+    "smallvec"
```
Comment (underfin) [inline .vscode/settings.json]: But it already merged into main branch

Comment (hyf0) [reply .vscode/settings.json]: Done

**Review by underfin — APPROVED** [review]

**Comment by hyf0** [comment]
### Merge activity

* **Nov 8, 11:30 PM**: @hyf0 started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/rolldown-rs/rolldown/194).
* **Nov 8, 11:31 PM**: @hyf0 merged this pull request with [Graphite](https://app.graphite.dev/github/pr/rolldown-rs/rolldown/194).

**Pull request merged**